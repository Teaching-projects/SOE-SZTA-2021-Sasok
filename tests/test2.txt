500
16
5
50
20
1
70
500
17
3