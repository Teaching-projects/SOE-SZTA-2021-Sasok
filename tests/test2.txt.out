Első egység HP-ja: Első egység DMG-je: Első egység AS-je: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A Csata elkezdődött! Harcos és Sámán megtámadták egymást. Életük: Harcos 483, Sámán 484
Sámán megtámadta Harcos, így Harcos élete - 466
Harcos megtámadta Sámán, így Sámán élete - 468
Sámán megtámadta Harcos, így Harcos élete - 449
Sámán megtámadta Harcos, így Harcos élete - 432
Harcos megtámadta Sámán, így Sámán élete - 452
Sámán megtámadta Harcos, így Harcos élete - 415
Harcos megtámadta Sámán, így Sámán élete - 436
Sámán megtámadta Harcos, így Harcos élete - 398
Sámán megtámadta Harcos, így Harcos élete - 381
Harcos megtámadta Sámán, így Sámán élete - 420
Sámán megtámadta Harcos, így Harcos élete - 364
Sámán megtámadta Harcos, így Harcos élete - 347
Harcos megtámadta Sámán, így Sámán élete - 404
Sámán megtámadta Harcos, így Harcos élete - 330
Harcos megtámadta Sámán, így Sámán élete - 388
Sámán megtámadta Harcos, így Harcos élete - 313
Sámán megtámadta Harcos, így Harcos élete - 296
Harcos megtámadta Sámán, így Sámán élete - 372
Sámán megtámadta Harcos, így Harcos élete - 279
Sámán megtámadta Harcos, így Harcos élete - 262
Harcos megtámadta Sámán, így Sámán élete - 356
Sámán megtámadta Harcos, így Harcos élete - 245
Harcos megtámadta Sámán, így Sámán élete - 340
Sámán megtámadta Harcos, így Harcos élete - 228
Sámán megtámadta Harcos, így Harcos élete - 211
Harcos megtámadta Sámán, így Sámán élete - 324
Sámán megtámadta Harcos, így Harcos élete - 194
Sámán megtámadta Harcos, így Harcos élete - 177
Harcos megtámadta Sámán, így Sámán élete - 308
Sámán megtámadta Harcos, így Harcos élete - 160
Harcos megtámadta Sámán, így Sámán élete - 292
Sámán megtámadta Harcos, így Harcos élete - 143
Sámán megtámadta Harcos, így Harcos élete - 126
Harcos megtámadta Sámán, így Sámán élete - 276
Sámán megtámadta Harcos, így Harcos élete - 109
Sámán megtámadta Harcos, így Harcos élete - 92
Harcos megtámadta Sámán, így Sámán élete - 260
Sámán megtámadta Harcos, így Harcos élete - 75
Harcos megtámadta Sámán, így Sámán élete - 244
Sámán megtámadta Harcos, így Harcos élete - 58
Sámán megtámadta Harcos, így Harcos élete - 41
Harcos megtámadta Sámán, így Sámán élete - 228
Sámán megtámadta Harcos, így Harcos élete - 24
Sámán megtámadta Harcos, így Harcos élete - 7
Harcos megtámadta Sámán, így Sámán élete - 212
Sámán megtámadta Harcos, így Harcos élete - -10

Sámán GYOZEDELMESKEDETT!
